network/evpn-gw/evpn-gw-cni/NOTICE

```diff
@@ -0,0 +1,12 @@
+EVPN GW CNI
```

Comment:
this file is not needed, what is the purpose ?

Reply:
The purpose is to have proper copyrights and give recognition for some parts of the code that have been copied from other repos that are under Apache2 License